hooks/useGovernanceAssets.ts

```diff
@@ -850,6 +854,57 @@ export default function useGovernanceAssets() {
       isVisible: canUseAuthorityInstruction,
       packageId: PackageEnum.VsrPlugin,
     },
+    // UXD PROTOCOL
```

Comment:
Can you replace this comment with:

https://patorjk.com/software/taag/#p=display&f=ANSI%20Regular&t=UXD%20PROTOCOL

please?